frontend/src/app/config.json

```diff
@@ -1,6 +1,7 @@
 {
     "patchFile": "/fe/config.patch.json",
     "inputThrottle": 300,
+    "eviction_type": "LRU",
```

Comment:
The console (frontend) config file is used for configuration related to visual settings. I don't think `eviction_type` should be added here.

Another small note,  the FE code conventions dictates the usage of camelcase and not snakecase

Reply:
irrelevant in latest changes